src/app/container/versions/versions.component.ts

```diff
@@ -52,8 +52,16 @@ export class VersionsContainerComponent extends Versions implements OnInit {
     });
   }
 
+  isManualMode() {
+    if (this.tool && this.tool.mode === DockstoreTool.ModeEnum.MANUALIMAGEPATH) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
   setNoOrderCols(): Array<number> {
-    return [ 5, 6 ];
+    return [5, 6];
```

Comment:
Not a focus of this PR, but this seems likely to break at some point as new columns are added and removed

Reply:
Not actually sure what this does.  Workflow doesn't have it, just tools.

Reply:
Does anything break if its removed? Worth checking out in a subsequent branch

Reply:
I removed it at some point and at a glance did not notice any significant changes.  Will definitely take a closer look later.

Reply:
setNoOrderCols makes the verified and action columns unsortable and there are different columns in the workflows vs tools.  So it does look necessary.